Sources/Penny/Handlers/MessageDeleteHandler.swift

```diff
@@ -41,29 +44,53 @@ struct MessageDeleteHandler {
     }
 }
 
-extension Payloads.CreateMessage {
+private extension Payloads.CreateMessage {
     init(
         messageCreate: Gateway.MessageCreate,
         author: DiscordUser
     ) {
-        let avatarURL = author.avatar.map {
-            CDNEndpoint.userAvatar(
-                userId: author.id,
-                avatar: $0
-            ).url
-        }
+        let member = messageCreate.member
+        let avatarURL = member?.uiAvatarURL ?? author.uiAvatarURL
+        let username = member?.uiName ?? author.username
+        let messageName = "message_\(messageCreate.id.rawValue)"
```

Comment:
intentionally doesn't contain the .json extension so Discord doesn't expand the text on Desktop which looks bad.